corehq/apps/data_interfaces/deduplication.py

```diff
@@ -67,7 +67,8 @@ def find_duplicate_case_ids(
     case_properties,
     include_closed=False,
     match_type="ALL",
-    case_filter_criteria=None
+    case_filter_criteria=None,
+    exclude_copied_cases=True,
```

Comment:
Correct. Not being used at present. I added it considering need for it in the future.  For e.g - In deduplication report - having an option like `Include Copied Cases` in the UI.
I would prefer to keep it, however absolutely fine to remove it in case it makes things less complex for the redesign work.

Reply:
I left the code as is, but I think we probably want to avoid adding code to handle things that might be necessary in the future. Much of the time, it ends up being a case of [YAGNI](https://martinfowler.com/bliki/Yagni.html), and it makes the code harder to reason about and maintain.

In this specific case, for example, if we wanted to include an option to include copied cases in the UI, we'd also need to modify `when_case_matches`, as with these changes, no copied duplicates are being recorded in the database.

Reply:
Good point regarding `YAGNI` and will keep a note of it for future.
One thing I forgot to mention is that `find_duplicate_case_ids` is supposed to have the responsibility of finding duplicate cases irrespective of how the cases were generated. Without the parameter, it would always exclude the copied cases and the consumer of this function would not explicitly know about it without looking at source code.